drivers/watchdog/wdt_iwdg_stm32.h

```diff
@@ -28,6 +28,10 @@ struct iwdg_stm32_config {
 struct iwdg_stm32_data {
 	uint32_t prescaler;
 	uint32_t reload;
+	/* IWDG user defined callback on EWI */
+	wdt_callback_t callback;
 };
 
+#define IWDG_STM32_STRUCT(dev) (((const struct iwdg_stm32_config *)dev->config)->instance)
```

Comment:
Those (`WWDG_STM32_STRUCT` and friends ) should actually be removed rather than added.
They were all over but were cleaned up, but wwdg felt through the cracks, as it seems.